core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -82,112 +86,19 @@ abstract class BaseConsumerTest extends IntegrationTestHarness with Logging {
   }
 
   @Test
-  def testAutoCommitOnRebalance() {
```

Comment:
Absolutely no objections. The tests are getting quite lengthy for my tastes and there's no point keeping redundant tests around.